deployment/v3/external/kafka/README.md

```diff
@@ -8,6 +8,89 @@
 $ ./install.sh
 ```
 
+## Steps to Backup & Restore Bitnami Kafka manually
+
+#### Prerequisites
+* Install `zip` & `unzip` packages on both backup/restore control plane nodes.
+
+#### Backup
+* Set WebSub and Kafka replicas to zero via the below command.
+  ```
+  kc2 -n default scale --replicas=0 deploy consolidator-websub-service websub-service
```

Comment:
use kubectl with namespace and kubeconfig.
This is because this is a generic process for backup. 
Or please define here what is kc2.